src/ui/components/pages/assets/assetItem.tsx

```diff
@@ -86,7 +86,17 @@ export function AssetItem({
         </div>
 
         <div className={styles.balance}>
-          <Balance isShortFormat={false} split={true} balance={balance} />
+          {!isLoading ? (
```

Comment:
Is it necessary? Asset loading check is done inside the `Balance` component

Reply:
This worked incorrectly — when there is no list of assets, a `PLACEHOLDER` is passed, where the array index is used as the `assetId`, which caused side work inside `Balance` component

Reply:
https://github.com/wavesplatform/waves-keeper/blob/KEEP-277/add-usd-amount/src/ui/components/ui/balance/Balance.tsx#L56:L57

In this case, we will see loader.
This logic does the same thing as `isLoading`.
When we hit `assets[1]`, undefined is returned and the loader is displayed.

Reply:
`assetId` isn't passed into `Balance` component here, so the prop defaults to `'WAVES'`. And for `'WAVES'` there's no request sent to the node, because it's info is hard-coded inside `AssetInfoController`.

Reply:
So the only side effect of this is that the request is sent from popup to the background and it stops there. But may be it makes sense to add a loader here directly to avoid all of that

Reply:
> So the only side effect of this is that the request is sent from popup to the background and it stops there. But may be it makes sense to add a loader here directly to avoid all of that

I think this is right solution.